tool_offset:
  translation: [0.0, 0.0, 0.077]
  rotation: [0.0, 0.0, 0.0, 1.0]
